docs/streams/upgrade-guide.html

```diff
@@ -34,9 +34,17 @@ <h1>Upgrade Guide and API Changes</h1>
     </div>
 
     <p>
-        If you are using Java 7 and want to upgrade from 1.0.x to 1.1.0 you don't need to make any code changes as the public API is fully backward compatible.
-        If you are using Java 8 method references in your Kafka Streams code you might need to update your code to resolve method ambiguties.
-	Hot-swaping the jar-file only might not work for this case.
+        If you want to upgrade from 1.1.x to 1.2.0 and you have customized window store implementations on the <code>ReadOnlyWindowStore</code> interface
+        you'd need to update your code to incorporate the newly added public APIs; otherwise you don't need to make any code changes.
+        See <a href="#streams_api_changes_120">below</a> for a complete list of 1.2.0 API and semantic changes that allow you to advance your application and/or simplify your code base.
+    </p>
+
+    <p>
+        If you want to upgrade from 1.0.x to 1.1.0 and you have customized window store implementations on the <code>ReadOnlyWindowStore</code> interface
+        you'd need to update your code to incorporate the newly added public APIs.
+        Otherwise, if you are using Java 7 you don't need to make any code changes as the public API is fully backward compatible;
+        byt if you are using Java 8 method references in your Kafka Streams code you might need to update your code to resolve method ambiguities.
+	    Hot-swaping the jar-file only might not work for this case.
```

Comment:
nit: seems your editor put a "tab" here

Reply:
Hmm that's weird, because on my intelliJ the lines are aligned.